var/spack/repos/builtin/packages/netcdf-c/package.py

```diff
@@ -141,6 +148,11 @@ class NetcdfC(AutotoolsPackage):
     conflicts("+parallel-netcdf", when="@:4.0")
     conflicts("+hdf4", when="@:4.0")
 
+    conflicts("+dap", when="+fismahigh")
```

Comment:
Should `+fismahigh` automatically switch those off? I think I am with you that it's better to warn the users, so that they know what's going on.

Reply:
My understanding from what @edwardhartnett said is that we want those three options off (correct me if I'm wrong). Since the variant is disabled by default, I'm assuming that users who enable it know what it is and want it to have that effect.

Reply:
Yes agreed. The only difference to what @edwardhartnett said is that I think we should only turn on `+fismahigh` on WCOSS2 or any other FISMA-high system, but nowhere else. Which means that it's good that the `byterange` variant is off by default, because `netcdf-c@4.9.1` doesn't compile with it (although they have a fix already, and presumably a 4.9.2 very soon). Until the compile error is fixed, I prefer `+dap +nczarr ~byterange` to be the default, and used on all our target systems except WCOSS2. Once the compile error is fixed, I prefer `+dap +nczarr +byterange` to be the default for all systems except WCOSS2.

Reply:
I'm not sure what change you're proposing, because I don't think these conflict statements will have any effect outside of WCOSS2. I don't think they stop us from setting `+dap +nczarr +byterange` (once the byte-range issue is fixed) in common/packages.yaml and applying those on all non-WCOSS2 systems, if that's what you mean. I'm certainly not proposing setting `+fismahigh` anywhere other than in our upcoming ops template.

Reply:
> I'm certainly not proposing setting +fismahigh anywhere other than in our upcoming ops template.
That's the short version of what I was saying :-)